dashboard/app/views/gallery_activities/_gallery_activities.html.haml

```diff
@@ -12,5 +12,5 @@
       %div.caption
         =t('gallery.caption_by_name_age', name: user.initial, age: user.age)
     %div.caption
-      =t('gallery.caption_time_ago', time_ago: time_ago_in_words(level_source.created_at))
+      =t('gallery.caption_time_ago', time_ago: time_ago_in_words(gallery_activity.created_at))
```

Comment:
I'd vote for `galley_activity.updated_at`, since that would be the same as `level_source.created_at`, right? (At least when it was a new level source).